dbt_meshify/dbt_projects.py

```diff
@@ -72,22 +75,27 @@ def __init__(
         )
 
     def _get_xproj_children_of_selected_nodes(self) -> Set[str]:
-        return {
-            model.unique_id
-            for model in self.models.values()
-            if any(parent for parent in model.depends_on.nodes if parent in self.resources)
-            and model.unique_id not in self.resources
-        }
+        all_children: Set[str] = set()
+        for resource in self.resources:
+            if resource.startswith("test"):
+                continue
+            all_children.update(self.child_map.get(resource, {}))
```

Comment:
I'm not sure if `{}` evaluates as an empty set, or an empty dict. Might want to confirm this.

Reply:
in fact, the maps return lists! updated the types accordingly, and confirmed that a list is an acceptable arg to the `Set.update()` method